src/crimson/os/seastore/onode_manager/staged-fltree/stages/stage_types.h

```diff
@@ -356,6 +356,12 @@ struct memory_range_t {
 
 enum class ContainerType { ITERATIVE, INDEXABLE };
 
+template <node_type_t> struct value_input_type;
```

Comment:
what is the difference between `value_input_type` and `value_type`? could you outline the difference in the comment of code?

Reply:
added comments inline.